### ISSUE TITLE
PyPi pacakge

### ISSUE THREAD

**Issue by Mattwmaster58** [k4m4/kickthemout#445]
Hi, I think you could really lower the friction to use this script if you crafted it into a Python pacakge and hosted it on PyPi. Then the installation instructions could simply be `pip install kickthemout` across all platforms.

**Comment by k4m4** [comment]
Yup, I've been thinking of creating a PyPi package for a while now. Thanks for your suggestion @Mattwmaster58!
